xrpl4j-core/src/test/java/org/xrpl/xrpl4j/model/transactions/json/PaymentFlagsJsonTests.java

```diff
@@ -45,35 +45,7 @@ public void testJson() throws JsonProcessingException, JSONException {
       .destination(Address.of("r4BPgS7DHebQiU31xWELvZawwSG2fSPJ7C"))
       .amount(XrpCurrencyAmount.ofDrops(25000000))
       .fee(XrpCurrencyAmount.ofDrops(10))
```

Comment:
Is it not necessary to test Payment without flags?

Reply:
Probably should. I'll put this back.